### PR TITLE
work/josh/render shop and profile

### PR DESCRIPTION
**Pull request by Joshua-Jack**
I put the shop and profile back into the top bar and took out the claim system from the shop.

### CODE REVIEW

File: src/components/topbar/profile/profile.tsx [hunk 1]
```diff
@@ -42,21 +42,23 @@ const Profile: FC = () => {
 
   return (
     <>
-      <img src={elon} alt="" className="profilepic" onClick={showOwner} />
-
+      {owned.isJust() && (
```
Comment (benefacto) [inline src/components/topbar/profile/profile.tsx:45]: Are we sure that we want to use this operator? Or do we only want to render the `img` if `owned.isJust()` is true? If so, we'd probably want something like `owned.isJust()?  : null` instead.

File: src/App.tsx [hunk 2]
```diff
@@ -23,24 +23,6 @@ function getLibrary(provider: any): Web3Provider {
 }
 function App() {
   const context = useWeb3React<Web3Provider>();
```
Comment (benefacto) [inline src/App.tsx:25]: Do we need this line or can it be removed or commented out for now?
```src/App.tsx
  Line 25:9:  'context' is assigned a value but never used  @typescript-eslint/no-unused-vars```

File: src/components/topbar/shop/shop.tsx [hunk 3]
```diff
@@ -1,10 +1,8 @@
 import { FC, useState, useMemo } from 'react';
```
Comment (benefacto) [inline src/components/topbar/shop/shop.tsx:1]: Can we remove/comment some unused variables in this file?
```
src/components/topbar/shop/shop.tsx
  Line 6:8:  '_' is defined but never used         @typescript-eslint/no-unused-vars
  Line 8:8:  'Outgoing' is defined but never used  @typescript-eslint/no-unused-vars
  Line 9:8:  'Claims' is defined but never used    @typescript-eslint/no-unused-vars
```

**Review by benefacto — CHANGES_REQUESTED** [review]
Would you mind taking a look at some of my comments? If you feel like they should be addressed, feel free to do so. Otherwise, we can handle them later. Either way, let me know when you're ready for this to be merged.

File: src/App.tsx (hunk 2 above)
Comment (Joshua-Jack) [reply src/App.tsx:25]: On it now

File: src/components/topbar/shop/shop.tsx (hunk 3 above)
Comment (Joshua-Jack) [reply src/components/topbar/shop/shop.tsx:1]: On it now

File: src/components/topbar/profile/profile.tsx (hunk 1 above)
Comment (Joshua-Jack) [reply src/components/topbar/profile/profile.tsx:45]: done

**Review by benefacto — APPROVED** [review]

**Pull request merged**